polaris-react/src/components/LegacyFilters/components/ConnectedFilterControl/ConnectedFilterControl.scss

```diff
@@ -122,4 +124,12 @@
 
 .AuxiliaryContainer {
   flex-grow: 0;
+
+  @media #{$p-breakpoints-md-down} {
+    margin-left: var(--p-space-200);
+  }
+
+  @media #{$p-breakpoints-md-up} {
+    margin-left: 0;
+  }
```

Comment:
Quick tophat and this update is functioning as expected 🙌

Only light suggestion I have is to rewrite this mobile first e.g.

```css
.AuxiliaryContainer {
  flex-grow: 0;
  margin-left: var(--p-space-200);

  @media #{$p-breakpoints-md-up} {
    margin-left: 0;
  }
}
```